packages/manager/apps/pci-cold-archive/src/pages/containers/edit-retention/EditRetention.page.tsx

```diff
@@ -100,12 +99,35 @@ export default function EditRetentionPage() {
 
   const onClose = onCancel;
 
+  const hasWarning = newRetentionDate < new Date(archive?.lockedUntil);
+  const error = {
+    min: lockedUntilDays < 1,
+    max: lockedUntilDays > MAX_RETENTION_DAYS,
+    nan: !RegExp(/^\d*$/).test(`${lockedUntilDays}`),
```

Comment:
I would rather move this constants in the `useMemo` function as testing a regex could be costly in terms of performance.